### DISCUSSION TITLE
En busca de un grupo

### DISCUSSION THREAD

**Discussion by miller-vrg** [Launch-X-Latam/MisionFrontEnd#552] [Team Posts]
Busco grupo mi correo es lx09903@innovaccion.mx

**Comment by R3D4N** [comment]
hola yo tampoco tengo grupo si quieres formamos team 😉 ,porsia mi correo es lx10972@innovaccion.mx

**Comment by NestorAMD** [comment]
Puedo ser con ustedes?

**Comment by HJavierMendez** [comment]
Me puedo unir a ustedes le pasó mi correo lx15369@innovaccion.mx

**Comment by NestorAMD** [comment]
Agréguenme mi correo es lx03359@innovaccion.mx

**Comment by oscararcee** [comment]
Me puedo unir a su equipo? Mi correo es lx02779@innovaccion.mx

**Comment by CrisQF** [comment]
Me puedo unir a su equipo? Mi correo es lx05668@innovaccion.mx

**Comment by fabiancgc12** [comment]
Todavia es posible unirse? mi correo es lx10623@innovaccion.mx
